src/TLuaInterpreter.cpp

```diff
@@ -14281,7 +14281,22 @@ void TLuaInterpreter::parseJSON(QString& key, const QString& string_data, const
         if (match.capturedStart() != -1) {
             QString title = match.captured(1);
             QString initialText = match.captured(2);
-            initialText.replace(QStringLiteral(R"(\n)"), QStringLiteral("\n"));
+            int j=0;
+            while((j = initialText.indexOf('\\', j)) != -1){
+                // while we find a backslash, look at the next character, decide what to replace them both with
+                switch(initialText.at(j+1).unicode()){
+                    case 'n' : initialText.replace(j,2,'\n'); break;
+                    case 't' : initialText.replace(j,2,'\t'); break; // IRE instead sends it out as \u0009
```

Comment:
`\t` is the representation of the ASCII single byte `\x09` so what is IRE sending out - `\u0009` might actually be the exact same thing... :confused: